src/main/java/org/jnode/fs/xfs/directory/ShortFormDirectoryEntry.java

```diff
@@ -64,7 +64,8 @@ public String getName() {
      * @return the inode number.
      */
     public long getINumber() {
-        int numberOffset = getNameLength() + 0x3;
+
+        int numberOffset = getNameLength() + this.inodeSize;
```

Comment:
If possible, only use `this` if the context of the variable call is not clear. Sometimes this is the case if you have a lambda function within a class where a variable within the lambda is the same as a member variable in the enclosing class.

Reply:
change applied.